protos/http/http.go

```diff
@@ -957,25 +955,28 @@ func (http *Http) hideHeaders(m *HttpMessage, msg []byte) {
 
 	if m.IsRequest {
 		// byte64 != encryption, so remove it from headers in case of Basic Authentication
-		auth_text := []byte("Authorization:")
-		if http.Strip_authorization && (m.Headers["authorization"] != "") {
-			header_len := m.bodyOffset - m.headerOffset
-			val_start_x := bytes.Index(msg[m.headerOffset:m.bodyOffset], auth_text)
-			val_end_x := -1
-			if val_start_x != -1 {
-				val_end_x = bytes.Index(msg[m.headerOffset+val_start_x:m.bodyOffset], []byte("\r\n"))
-
-				if val_end_x < 0 || val_end_x > header_len {
-					val_end_x = header_len
+		auth_text := []byte("uthorization:")   // [aA]
+		if http.Strip_authorization {
+
+			authheader_start_x := m.headerOffset + bytes.Index(msg[m.headerOffset:m.bodyOffset], auth_text)
```

Comment:
Can you use camelCase for the newly introduced variables? We are currently in the progress of refactoring all variable names. So all the new ones we introduce we already use the new naming scheme. Only change the ones that are local to your changes.